vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -949,6 +950,27 @@ public void failure(VimeoError error) {
         return call;
     }
 
+    /**
+     * Used to edit a users push notification subscriptions
+     *
+     * @param subscriptionMap a map of the subscriptions you wish to change
+     * @param callback        callback to be invoked when the request finishes
+     * @return A Call so that the request can be cancelled if need be
+     */
+    @Nullable
+    public Call<SubscriptionCollection> editSubscriptions(@NotNull Map<String, Boolean> subscriptionMap,
+                                                          @NotNull
+                                                                  ModelCallback<SubscriptionCollection> callback) {
+        if (callback == null) {
```

Comment:
Null check not needed if you are annotating. 

If you feel ambitious, you could change the other methods in this class that take a callback to have the  @NotNull annotation too and remove their null checks. Obviously that extra work is not required.